### PR TITLE
Issue 14732: Fix EmptyTokenTest for MP JWT TCK

### LINKED ISSUE #14732: MP JWT 1.2 issue with EmptyTokenTest methods
**Describe the bug**
We have a bit of a catch-22 with the methods in the EmptyTokenTest even with the latest MP JWT 1.2 TCK changes from master branch which includes a fix for media type with https://github.com/eclipse/microprofile-jwt-auth/pull/212.

The EmptyTokenTest consists of 3 test methods. 
1. With changes from master branch, the method `emptyToken` now passes with the expected RC 200.
2. The invalidToken test which expects a 401 RC is getting a 200 RC and is failing.

We were looking at the TAI property of `<trustAssociation invokeForUnprotectedURI="true" />`, but if I set this property then the emptyToken test fails and the invalidToken test passes.

EmptyTokenTest methods

```Java
   @RunAsClient
    @Test(groups = TEST_GROUP_JAXRS,
          description = "Validate that an empty JWT in injected in the endpoint")
    public void emptyToken() {
        String uri = baseURL.toExternalForm() + "endp/verifyEmptyToken";
        WebTarget echoEndpointTarget = ClientBuilder.newClient().target(uri);
        Response response = echoEndpointTarget.request(TEXT_PLAIN).get();
        Assert.assertEquals(response.getStatus(), HttpURLConnection.HTTP_OK);
        String replyString = response.readEntity(String.class);
        JsonReader jsonReader = Json.createReader(new StringReader(replyString));
        JsonObject reply = jsonReader.readObject();
        Assert.assertTrue(reply.getBoolean("pass"));
    }

    @RunAsClient
    @Test(groups = TEST_GROUP_JAXRS,
          description = "Validate that a token sent to an unauthenticated / unauthorized endpoint is verified")
    public void invalidToken() {
        String uri = baseURL.toExternalForm() + "endp/verifyEmptyToken";
        WebTarget echoEndpointTarget = ClientBuilder.newClient()
                                                    .target(uri)
                                                    .queryParam("input", "hello");
        Response response = echoEndpointTarget
            .request(TEXT_PLAIN)
            .header(HttpHeaders.AUTHORIZATION, "Bearer " + "something")
            .get();
        Assert.assertEquals(response.getStatus(), HttpURLConnection.HTTP_UNAUTHORIZED);
    }

    @RunAsClient
    @Test(groups = TEST_GROUP_JAXRS,
          description = "Validate that a token sent to an unauthenticated / unauthorized endpoint is verified and " +
                        "injected as non-empty")
    public void validToken() throws Exception {
        String token = TokenUtils.generateTokenString("/Token1.json");
        String uri = baseURL.toExternalForm() + "endp/verifyNonEmptyToken";
        WebTarget echoEndpointTarget = ClientBuilder.newClient()
                                                    .target(uri)
                                                    .queryParam("input", "hello");
        Response response = echoEndpointTarget
            .request(TEXT_PLAIN)
            .header(HttpHeaders.AUTHORIZATION, "Bearer " + token)
            .get();
        Assert.assertEquals(response.getStatus(), HttpURLConnection.HTTP_OK);
        String replyString = response.readEntity(String.class);
        JsonReader jsonReader = Json.createReader(new StringReader(replyString));
        JsonObject reply = jsonReader.readObject();
        Assert.assertTrue(reply.getBoolean("pass"));
    }
```

**Steps to Reproduce**
Steps to reproduce the bug

**Expected behavior**

**Diagnostic information:**
 - OpenLiberty Version: [e.g. 19.0.0.8]
 - Java Version: [i.e. full output of `java -version`]
 - server.xml configuration (WITHOUT sensitive information like passwords)
 - If it would be useful, upload the messages.log file found in `$WLP_OUTPUT_DIR/messages.log`

**Additional context**

### PR DESCRIPTION
**Pull request by ayoho**
Fixes #14732

### CODE REVIEW

**Comment by ayoho** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.mp.jwt.1.1_fat,com.ibm.ws.security.mp.jwt.1.2_fat

**Comment by ayoho** [comment]
#libby

**Comment by ayoho** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.mp.jwt.1.1_fat,com.ibm.ws.security.mp.jwt.1.2_fat

File: dev/com.ibm.ws.security.mp.jwt/src/com/ibm/ws/security/mp/jwt/tai/MicroProfileJwtTAI.java [hunk 1]
```diff
@@ -373,6 +370,25 @@ TAIResult getAndValidateMicroProfileJwt(HttpServletRequest request, HttpServletR
         return result;
     }
 
+    TAIResult getResultForMissingToken(HttpServletRequest request, HttpServletResponse response, String methodName) throws WebTrustAssociationFailedException {
```
Comment (ayoho) [inline dev/com.ibm.ws.security.mp.jwt/src/com/ibm/ws/security/mp/jwt/tai/MicroProfileJwtTAI.java:373]: `String methodName` got brought in by my IDE when extracting this code to a separate method; it's not needed and should be removed in a followup.

**Review by teddyjtorres — APPROVED** [review]

**Pull request merged**